Movie_MVP/Movie_MVP/RealmService/RealmService.swift

```diff
@@ -0,0 +1,61 @@
+// RealmService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import RealmSwift
+
+/// Реалм сервис
+final class RealmService: RealmServiceProtocol {
+    func saveObject<T>(items: [T]) where T: Object {
+        do {
+            let realm = try Realm()
+            try realm.write {
+                realm.add(items, update: .modified)
+            }
+        } catch {
+            print(error.localizedDescription)
+        }
+    }
+
+    func getMovieTypeObjects<T: Object>(_ type: T.Type, movieType: MovieType) -> Results<T>? {
+        do {
+            let realm = try Realm()
+            let object = realm.objects(type).filter("movieType = %@", movieType.rawValue)
+            return object
+        } catch {
+            print(error.localizedDescription)
+        }
+        return nil
+    }
+
+    func getCastObjects<T: Object>(_ type: T.Type) -> Results<T>? {
+        do {
+            let realm = try Realm()
+            return realm.objects(type)
+        } catch {
+            print(error.localizedDescription)
+        }
+        return nil
+    }
+
+    func getObject<T: Object>(_ type: T.Type, id: Int) -> T? {
+        do {
+            let realm = try Realm()
+            return realm.object(ofType: type, forPrimaryKey: id)
+        } catch {
+            print(error.localizedDescription)
+        }
+        return nil
+    }
+
+    func deleteObject<T: Object>(_ type: T) {
```

Comment:
```suggestion
    func deleteObject<T: Object>(_ type: T) {
```
этот метод нигде больше используется, он точно нужен?